client-js/app/elements/labrad-plot.ts

```diff
@@ -95,6 +95,9 @@ export class Plot extends polymer.Base {
   @property({type: Number, value: 0})
   numIndeps: number;
 
+  @property({type: String, value: ''})
+  currPos: string;
```

Comment:
I see. I would say keep it as a string, but I would move the expository text to the html template.